Model/BoldQuoteRepository.php

```diff
@@ -0,0 +1,37 @@
+<?php
+//phpcs:disable Magento2.Annotation.MethodArguments.NoCommentBlock
+//phpcs:disable Magento2.Annotation.MethodArguments.ParamMissing
+//phpcs:disable Magento2.Annotation.MethodAnnotationStructure.MethodAnnotation
+
+declare(strict_types = 1);
```

Comment:
Standards: [PER-CS](https://www.php-fig.org/per/coding-style/)/[PSR-12](https://www.php-fig.org/psr/psr-12/) state that `declare` statements must come first in PHP files.